extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -900,7 +900,7 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
    */
   getSlotSize() {
     const {width, height} = this.getDeclaredSlotSize_();
-    return width && height
+    return !isNaN(width) && !isNaN(height)
```

Comment:
Do you want to check for case where either width or height is 0?

Reply:
Actually, I don't need this anymore. Best to keep as is to avoid unintended side effects.